src/qs8-gemm/scalar-qd8-f16-qb4w.c.in

```diff
@@ -0,0 +1,163 @@
+// Copyright 2021 Google LLC
```

Comment:
Do you want to merge this in with `scalar.c.in`? Or we have to explain why this is outside. I am fine with either.

Reply:
Yeah, I do prefer that. It may be a bit more work to do cleanly since that meta kernel doesn't have fp16 support, but I think it makes more sense to do this.

Reply:
Done.